pkgs/games/scarab/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, dotnetCorePackages
+, glibc
+, zlib
+, libX11
+, libICE
+, libSM
+, fontconfig
+, gtk3
+, copyDesktopItems
+, graphicsmagick
+, wrapGAppsHook
+, makeDesktopItem
+}:
+
+buildDotnetModule rec {
+  pname = "scarab";
+  version = "1.19.0.0";
+
+  src = fetchFromGitHub {
+    owner = "fifty-six";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "10pmzy7nhcam0686kpn64cgar59shzzy5k7j3vrgmnm27zgkr22v";
+  };
+
+  nugetDeps = ./deps.nix;
+  projectFile = "Scarab.sln";
+  executables = [ "Scarab" ];
+
+  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+
+  runtimeDeps = [
+    glibc
+    zlib
+    libX11
+    libICE
+    libSM
+    fontconfig
+    gtk3
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    graphicsmagick
+    wrapGAppsHook
+    gtk3
+  ];
+
+  postFixup = ''
+    # Icon for .desktop
+    install -d -m755 $out/share/icons/hicolor/256x256/apps/
```

Comment:
Shouldn't this be an `mkdir` instead? The icon probably doesn't need these permissions.

Reply:
Yeah I forgot that those permissions should be the default for `mkdir`.